roles/kubernetes-apps/ingress_controller/cert_manager/templates/cert-manager.yml.j2

```diff
@@ -964,6 +976,18 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+{% if cert_manager_http_proxy is defined and cert_manager_http_proxy != "" %}
+          - name: HTTP_PROXY
+            value: "{{ cert_manager_http_proxy }}"
+{% endif %}
+{% if cert_manager_https_proxy is defined and cert_manager_https_proxy != "" %}
+          - name: HTTPS_PROXY
+            value: "{{ cert_manager_https_proxy }}"
+{% endif %}
+{% if cert_manager_no_proxy is cert_manager_no_proxy != "" %}
```

Comment:
Guessing that you missed `is defined and` here?

Reply:
My mistake. I will fix it.